EIPS/eip-191.md

```diff
@@ -55,29 +54,55 @@ Using `0x19` thus makes it possible to extend the scheme by defining a version `
 |    `0x01`    | [712][eip-712] | Structured data
 |    `0x45`    | [191][eip-191] | `personal_sign` messages
 
+#### Version `0x00`
+
+```
+0x19 <0x00> <intended validator address> <data to sign>
+```
+
+The version `0x00` has `<intended validator address>` for the version specific data. In the case of a Multisig wallet that perform an execution based on a passed signature, the validator address is the address of the Multisig itself. The data to sign could be any arbitrary data.
+
+#### Version `0x01`
+
+The version `0x01` is for structured data as defined in [EIP-712]
```

Comment:
QQ: Should we mention anything about the EIP-712 "domain" besides the "structured data" here?

Reply:
@xinbenlv  According to @holiman the co-author of the EIP191 standard:
> I'd prefer to leave the definition fully to EIP-712. So simply say:
The version 0x01 is for structured data as defined in [EIP-712](https://github.com/ethereum/EIPs/pull/eip-712)